src/integrationTest/resources/TestProject/build.gradle

```diff
@@ -8,9 +8,8 @@ buildscript {
   dependencies {
```

Comment:
Do we still need the `integrationTest` projects as we use the project via the gradle test runner?

Reply:
I think we could convert it from the current style to the gradle test runner style we use for everything else (e.g. have the build file included within CompilationAndGeneration.groovy). I'm not 100% sure how it would work, though, since it's one of the more complicated tests (there are subprojects with their own build files, etc). I'd say it would be a "nice to have" feature to have a more consistent testing approach, but that it's probably not worth the effort to switch it over.

Reply:
And to be clear, I don't think we use the `integrationTest` project outside of `CompilationAndGeneration.groovy`.

Reply:
Yeah, I don't think its necessary -- we should be able to use the groovy test runner + temp folders.